ExamenProeveUnity/Assets/Scripts/Runtime/Managers/PlayerManager.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine.InputSystem;
+using Utilities.MethodExtensions;
+using Utilities.Other.Runtime;
+
+namespace Runtime.Managers
+{
+    public class PlayerManager : MonoSingleton<PlayerManager>
+    {
+        private List<Entity> _players = new();
+        private List<PlayerInput> _playerInputs = new();
+
+        protected override void Awake()
+        {
+            base.Awake();
+            _playerInputs = FindObjectsOfType<PlayerInput>().ToList();
+            foreach (var playerInput in _playerInputs)
+            {
+                _players.Add(playerInput.GetComponent<Entity>());
```

Comment:
wat als er geen entity script is (we kunnen er van uit gaan dat het er is maar mochten we op alles letten zou dit een ding geweest zijn)